pages/docs/reference/coding-conventions.md

```diff
@@ -726,6 +727,253 @@ foo {
+
+</div>
+
+Using trailing commas has several benefits:
+
+* It makes version-control diffs cleaner: all focus is on changed value.
+* It's easy to add or reorder elements: no need to add or delete the comma sign if you manipulate elements.
+* It simplifies code generation (for example, for object initializers). The last element can also have the comma sign.
+
+Using trailing commas are optional: your code will work without them. But Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it optional for the call site.
+
+To enable trailing commas for the IntelliJ formatter, go to __Settings | Editor | Code Style | Kotlin__, 
+open the __Other__ tab and enable the __Use trailing comma__ option.
+
+Kotlin supports trailing commas in the following cases:
+* [Lists, enumerations, value arguments](#lists-enumerations-value-arguments)
+* [Class parameters](#class-parameters)
+* [Function value parameters](#function-value-parameters)
+* [Parameters with optional type (including setters)](#parameters-with-optional-type-including-setters)
+* [Indexing suffix](#indexing-suffix)
+* [Lambda parameters](#lambda-parameters)
+* [`when` entry](#when-entry)
+* [Collection literals (in annotations)](#collection-literals-in-annotations)
+* [Type arguments and type parameters](#type-arguments-and-type-parameters)
+* [Destructuring declarations](#destructuring-declarations)
+
+#### Lists, enumerations, value arguments
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+```kotlin
+val colors = listOf(
+    "red",
+    "green",
+    "blue", // trailing comma
+)
+
+enum class Direction {
+    NORTH,
+    SOUTH,
+    WEST,
+    EAST, // trailing comma
+}
+
+fun sum(x: Any, y: Any) {}
+
+fun main() {
+    sum(
+        10,
+        20, // trailing comma
+    )
+}
+```
+
+</div>
+
+#### Class parameters
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+```kotlin
+class Customer(
+    val name: String,
+    val surname: String, // trailing comma
+)
+```
+
+</div>
+
+#### Function value parameters
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+```kotlin
+fun powerOf(
+    number: Int, 
+    exponent: Int, // trailing comma
+) { /*...*/ }
+
+constructor(
+    x: Comparable<Number>,
+    y: Iterable<Number>, // trailing comma
+) {}
+
+fun print(
+    vararg quantity: Int,
+    description: String, // trailing comma
+) {}
+```
+
+</div>
+
+#### Parameters with optional type (including setters)
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+```kotlin
+val sum: (Int, Int, Int) -> Int = fun(
+    x,
+    y,
+    z, // trailing comma
+): Int {
+    return x + y + x
+}
+println(sum(8, 8, 8))
+```
+
+</div>
+
+#### Indexing suffix
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+```kotlin
+class Surface {
+    operator fun get(x: Int, y: Int) = 2*x + 4*y - 10
```

Comment:
Kotlin coding conventions (aka this page :) ) says [put spaces around binary operators (a + b).](https://kotlinlang.org/docs/reference/coding-conventions.html#horizontal-whitespace)